tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -1,8 +1,10 @@
+from datetime import datetime, timezone
+
 import pandas as pd
-import sqlalchemy.dialects.postgresql as POSTGRESQL_TYPES
```

Comment:
we shouldn't import directly from sqlalchemy

Reply:
Note to non-Josh folks: the postgresql types were just imported from the sqla types, so they are equivalent. Which explains mypy being fine with this change.